roles/openshift_master/templates/v1_partials/oauthConfig.j2

```diff
@@ -77,6 +77,8 @@ oauthConfig:
   - name: {{ identity_provider.name }}
     challenge: {{ identity_provider.challenge }}
     login: {{ identity_provider.login }}
+{% set default_mapping_method = 'add' if openshift.master.identity_providers | length > 1 else 'claim' %}
+    mappingMethod: {{ identity_provider.mappingMethod | default(default_mapping_method) }}
```

Comment:
I would probably call out the default mapping method in the file that drives ansible, with a comment explaining the implications. I would maaaybe still default it to claim... not sure